### DISCUSSION TITLE
Impossible to clone the repository and download the zip file

### DISCUSSION THREAD

**Discussion by cristiancarrino** [community/community#88694] [Repositories]
### Select Topic Area

Bug

### Body

I got an error when I try to clone my progect from the cli. The error says: 
"error: RPC failed; curl 92 HTTP/2 stream 0 was not closed cleanly: CANCEL (err 8)
error: 7811 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output"

Also if I download the zip file of the progect from GitHub website, when I try to open it a get this error: "Unespected end of file".

It is impossible for me to recover the project

**Comment by AidenJamesYt** [comment]
from what i see, you might have a problem with the network connection or the integrity of the data being transferred

**Reply by AidenJamesYt** [reply]
you are getting errors related to RPC failure and unexpected disconnects. io troubleshoot, make sure that you have a stable internet connection, retry cloning, switch from SSH to HTTPS, increase buffer size, check disk space, and verify your Gitversion. if cloning persists, download the ZIP file from the GitHub website. make sure your Git version is up-to-date and consider proxy configuration if behind a proxy

**Reply by cristiancarrino** [reply]
Thank you for your tips.
Just a recap of my situation: 
1. I am sure is not a connection issue because I didn't get any error during download and because in the same moment I was downloading other files that I can open successfully. Also I tried several times to download zip file and clone project, maybe 5-6 times from two different houses with two different connections.
2. I was cloning from HTTPS
3. disk space is enought
4. my git version is the 2.34.1
5. I don't have any proxy (I am at home with no proxy)

Do you have other suggestions? I think it is a GitHub's issue
